lib/ansible/modules/windows/win_iis_webapppool.py

```diff
@@ -58,40 +58,118 @@
 '''
 
 EXAMPLES = r'''
-- name: Return information about an existing application pool
-  win_iis_webapppool:
-    name: DefaultAppPool
+# This returns information about an existing application pool
```

Comment:
Yes I did, and that is actually what the [Documenti page](http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html) says ("Examples should be written in YAML format in plain text in an EXAMPLES string within the module")